### DISCUSSION TITLE
Webhook -Hostinger Invalid HTTP response : 400

### DISCUSSION THREAD

**Discussion by webmanagementhk** [community/community#50526] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I recently tried to deploy my repository on Hostinger without success. 

- I followed the guide https://support.hostinger.com/en/articles/1583302-how-to-deploy-a-git-repository multiple times trying to figure out what was the problem. I cleared cache, flush dns, delete file manager files on hostinger and started again.
- I reached Hostinger support 3 times and they are lost, basically everything seems fine on their side but Github is unable to deliver.
- Invalid HTTP response : 400
- Also im unable to load the website online.

I cant find anything related to this problem and i hope im not the first one... thank you for your support

**Comment by SteveNyanumba** [comment, marked as answer]
Your problem is on the Github webhooks side. Just change the content type from application/x-www-form-urlencoded to application/json and see if that helps.

**Reply by apsmono** [reply]
oh god, fortunately, someone has the same problem as me.  this one solved my problem. thanks!!

**Reply by webmanagementhk** [reply]
Aleluya!!!! I thought about it!!!! but it didnt make sense... thank you bro!! Forever grateful!
